static_src/components/app_container.jsx

```diff
@@ -108,6 +111,28 @@ export default class AppContainer extends React.Component {
     );
   }
 
+  get openApp() {
+    if (AppStore.isRunning(this.state.app)) {
```

Comment:
Dont' we only want the button if the app is actually running? Otherwise we'd be sending a user to a broken URL?